src/main/resources/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0

```

Comment:
Should this file be checked in?  I figured it would be generated.

Reply:
checked in one is only for ide debugging - runtime is always generated

Reply:
just update the email addr below and we can merge this request.  we have tickets to track the items of concern.